src/test/java/org/example/Operations/ResourceOperationsTest.java

```diff
@@ -53,6 +54,7 @@ public void setup() {
         aliceToken = alice.getAuthToken();
     }
 
+    @Ignore
```

Comment:
I think you can include a message with this.
```java
@Ignore("Remove this annotation once issue #N is resolved")
```
Where `N` is the issue number.

Reply:
Done.